src/components/design-system.css

```diff
@@ -161,14 +161,16 @@ textarea {
 
 /* @getethos/design-system/Images.scss
    ========================================================================== */
+/* Generally used as a wrapper around Image components. */
 figure {
   margin: 0;
 }
 
 .Image {
   display: block;
+  max-width: 100%;
   width: 100%;
-  height: auto;
+  object-fit: cover;
```

Comment:
It would be nice if this stuff could be true CSS Modules

Reply:
I will try converting this to css modules

Reply:
Ok, I'm fine if you'd like to do on a follow up though. Up to you

Reply:
converted